fabfile/fabfile.py

```
@@ -138,8 +138,23 @@ def deploy():
     """
     Deploy Kirin services
     """
+    print_status()
     execute(deploy_kirin)
     execute(deploy_kirin_beat)
+    print_status()
+
+
+def print_status():
+
+    request = 'http://{}/status'.format(env.kirin_host)
+    resp = check_node(request)
+    print("")
+    print("curl {}".format(request))
+    if resp:
+        print(resp.json())
+    else:
+        print("[unavailable]")
```

Would it make sense to deploy Kirin is the host is unavailable ?! 
Should we throw an exception here ? or wait and retry until it's available ?

---

Well it's not perfect, but yes the solution would be retrying...
I can change that, and we'll just lose the side effect of correction (still, stopping earlier is better :wink: )

---

And currently, it mostly means slow, more than unavailable :disappointed:

---

Added retrying, tried to check what can cause the RetryError on dev.
Looks like it's not timeout nor 502, could not reproduice using httpbin.org :-|